Makefile.coq_modules

```diff
@@ -43,14 +43,15 @@ MODULES = \
   ErgoC/Lang/ErgoCStdlib \
   ErgoC/Lang/ErgoCEvalContext \
   ErgoC/Lang/ErgoCEval \
+  ErgoC/Lang/ErgoCTypeContext \
+  ErgoC/Lang/ErgoCType \
   ErgoNNRC/Lang/ErgoNNRC \
   ErgoNNRC/Lang/ErgoNNRCSugar \
   Translation/CTOtoErgo \
   Translation/ErgoNameResolve \
   Translation/ErgoCompContext \
   Translation/ErgoCInline \
   Translation/ErgotoErgoC \
-  Translation/ErgoCtoErgoNNRCType \
```

Comment:
Ouch! Please put this one back (unless I misunderstand the intent) since this is the small start of #329